chatsky/core/context.py

```diff
@@ -147,6 +147,14 @@ class Context(BaseModel):
     It is meant to be used by the framework only. Accessing it may result in pipeline breakage.
     """
 
+    @property
+    def origin_interface(self) -> Optional[str]:
+        if 1 in self.requests.keys():
+            origin = self.requests[1].origin
```

Comment:
Wouldn't that require loading the first request from the db in case of partial context storage?
What if we pass first request along with start label in the `init` method and set `origin_interface` there?